### PR TITLE
Update gettingstarted.rst

### PR DESCRIPTION
**Pull request by marvelm**
I replaced DDL with SchemaDescription word for word.

### CODE REVIEW

**Comment by typesafehub-validator** [comment]
Hi @marvelm,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Typesafe Contributors License Agreement:

[http://www.typesafe.com/contribute/cla](http://www.typesafe.com/contribute/cla)

**Comment by cvogt** [comment]
LGTM

**Comment by marvelm** [comment]
You should merge this solely because of the PR number.

**Comment by cvogt** [comment]
I agree ;)

File: slick/src/sphinx/gettingstarted.rst [hunk 1]
```diff
@@ -142,9 +142,9 @@ data:
 
 .. includecode:: code/FirstExample.scala#create
 
-The ``TableQuery``'s ``ddl`` method creates ``DDL`` (data definition language) objects
-with the database-specific code for creating and dropping tables and other
-database entities. Multiple ``DDL`` values can be combined with ``++`` to
+An implicit conversion of ``TableQuery`` gives us an object with driver-specific extension methods.
```
Comment (szeiger) [inline slick/src/sphinx/gettingstarted.rst]: Terminology has changed in 3.2. Replace "driver-specific" by "profile-specific".

**Comment by szeiger** [comment]
This should also be squashed so we don't get 3 commits for a 3-line doc change. But I agree, then we need to merge #1337 :)

**Comment by cvogt** [comment]
@marvelm you mind updating this as requested :)?

**Comment by nafg** [comment]
Meanwhile the docs were converted to Ornate and this file was obsolete, so now we have a gettingstarted.rst for no reason

**Pull request merged**